.travis.yml

```diff
@@ -0,0 +1,201 @@
+---
+sudo: required
+dist: trusty
+
+services:
+  - docker
+
+script:
+  # Build docker image
+  - cd 2.6/
+  - docker build -t maestrano/mongo:travis . > /dev/null 2>&1
+
+  # Create volumes
+  - mkdir -p vol-m0 vol-m1 vol-m2
+
+  # Configuration
+  - HOST_PUB_IP=$(ip route get 8.8.8.8 | awk '{print $NF; exit}')
+  - PORT_NODE_0=33000
+  - VOL_NODE_0=$(pwd)/vol-m0
+  - PORT_NODE_1=33001
+  - VOL_NODE_1=$(pwd)/vol-m1
+  - PORT_NODE_2=33002
+  - VOL_NODE_2=$(pwd)/vol-m2
+  - PORT_NODE_3=33005
+  - VOL_NODE_3=$(pwd)/vol-m3
+  - CLUSTER_CONFIG=$HOST_PUB_IP:$PORT_NODE_1:1,$HOST_PUB_IP:$PORT_NODE_2:2
+
+  #============================================================================
+  # Test standalone Mongo
+  #============================================================================
+  # Start standalone instance
+  - docker run -d -p $PORT_NODE_0:27017 -v $VOL_NODE_0:/data -e MONGO_USER=root -e MONGO_PASSWORD=root --name m0 maestrano/mongo:travis
+  - sleep 5
+
+  # Wait for standalone instance to be up
+  - for i in {30..0}; do mongo -u root -p root --quiet --eval 'rs.isMaster().ok' $HOST_PUB_IP:$PORT_NODE_0/admin &> /dev/null && break; sleep 1; done
```

Comment:
I will if we eventually move to a travis.sh script. For now I prefer to keep it simple.